### ISSUE TITLE
Missing 1.24 Ubuntu packages

### ISSUE THREAD

**Issue by hulquest** [alvistack/ansible-role-cri_o#3]
I work on a product that continues to support k8s and cri-o 1.24 and I would like to pin to tag 7.4.0 since 1.24 was deprecated earlier this week.  However the packages are missing from the apt repo.  Will you restore these versions for the [7.4.0 tag](https://github.com/alvistack/ansible-role-cri_o/blob/7.4.0/vars/main.yml)

**Comment by hswong3i** [comment]
Generally I will only support N/N-1/N-2 Kubernetes release, so as 1.27 released 1.24 is therefore being removed.

Please kindly understand that keep maintenance latest 3 releases across ~20 OS (even as old as CentOS 7 or Ubuntu 18.04) is not a easy job... I need to keep maintaining its dev toolset, e.g. golang 1.18, too.

**Comment by hulquest** [comment]
I completely understand and appreciate it!  Would you consider switching to the upstream apt repositories for kubelet and cri-o?  Then we can pin to a tag in this repo and override `_cri_o_version` if necessary?

**Comment by hswong3i** [comment]
Some more details: due to simplified implementation with Ansible templating, some config files, e.g. kubeadm.yml for bootstrapping k8s, or cri-o config file, are using the N-2 version as much as possible for backward compatibility.

BTW, some of these config files parameters are also discontinued when compare across N-3 vs N release, and it will break the setup. 

Therefore supporting up to N-2 is proofed by daily practice, where up to N-3 will not works.

**Comment by hulquest** [comment]
Understood.  Thank you, I'll close the issue.

**Comment by hswong3i** [comment]
> I completely understand and appreciate it!  Would you consider switching to the upstream apt repositories for kubelet and cri-o?  Then we can pin to a tag in this repo and override `_cri_o_version` if necessary?

Which upstream are you expecting?

If considering kubic repo, they couldn't support as much as OS with unified configuration experience (and that's why I keep hard working on my own repo under my own control).

If considering OS specific repo, usually only single kubernetes release supported. So again not a good choice.

If considering upstream static binary + our own complicated Ansible configuration templating, this is my old bad day method + a lot of nix-based static binary contributions to upstream, but proofed as time wasting when compare with OBS based packaging.

Long story short: current N-2 + OBS repo + simplified Ansible implementation working smoothly and reasonably.
